sql/src/main/java/io/crate/action/sql/DCLStatementDispatcher.java

```diff
@@ -35,11 +35,11 @@
 import io.crate.data.Row;
 import io.crate.exceptions.SQLExceptions;
 import io.crate.metadata.rule.ingest.IngestRule;
```

Comment:
any reasons not to move all the others like all of `io.crate.metadata.rule.ingest` and `IngestionRuleInfo`,  `IngestionRulesInfo`etc. ?

Reply:
It also handles privileges. Maybe we should split it up later on. Anyway, thanks for the pointer because I had missed the `DDLStatementDispatcher`